api/src/paths/access-request.ts

```diff
@@ -150,6 +158,8 @@ export function updateAccessRequest(): RequestHandler {
 
       await connection.commit();
 
+      sendApprovalEmail(administrativeActivityStatusTypeId, req['keycloak_token'], userIdentifier, identitySource);
```

Comment:
Move this before the connection.commit, and then just pass the connection to the function as a param.

Reply:
Then you dont need to get a new one, or open it, as it will already be ready to use